__tests__/app.test.js

```diff
@@ -93,4 +95,20 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(article.votes).toBe(90);
       });
```

Comment:
Its just missing a few tests

- [ ]  400 inc_vote not an integer e.g. {inc_votes: 'not a number'}

- [ ] 200/400 inc_vote key missing - either respond with an error or make no change.

Reply:
Not entirely sure what you mean by 200/400 inc_vote key missing - I've interpreted the best I could. I'm sure you'll let me know if I interpreted it incorrectly!